### PR TITLE
Fixed timeout error during mysql testing.

### PR DESCRIPTION
**Pull request by kshakir**
Tagged all database specific tests as `IntegrationTest`.
As mysql now has 5s to timeout, increased the testing timeout used to detect if mysql is available.
Removed unused code and optimized imports.

### CODE REVIEW

**Comment by kshakir** [comment]
Note for reviewers verifying testing changes: Travis, `sbt test`, and `sbt 'test-only cromwell.engine.db.slick.SlickDataAccessSpec'` no longer run the database integration tests. One must use either:

```bash
sbt 'integration:test-only cromwell.engine.db.slick.SlickDataAccessSpec'
```

or:

```bash
sbt 'alltests:test-only cromwell.engine.db.slick.SlickDataAccessSpec'
```

@cjllanwarne First reviewer.

File: src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala [hunk 1]
```diff
@@ -650,7 +649,6 @@ class SlickDataAccessSpec extends FlatSpec with Matchers with ScalaFutures with
       val workflowInfo = WorkflowDescriptor(workflowId, testSources)
       val task = Task.empty
       val call = new Call(None, callFqn, task, Set.empty[FullyQualifiedName], Map.empty, None)
-      val workflowOutputList = Seq(ReportableSymbol("call.fully.qualified.scope.symbol", WdlArrayType(WdlStringType)))
```
Comment (cjllanwarne) [inline src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala]: I guess this is highlighted as never used? But was this checking that no exceptions were thrown, etc?

**Comment by cjllanwarne** [comment]
:+1:

**Comment by cjllanwarne** [comment]
Also @scottfrazer

File: src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala (hunk 1 above)
Comment (kshakir) [inline src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala]: The value was orphaned during 968b115e071f54684dddee397149465b3f5a9f99.

**Comment by scottfrazer** [comment]
:+1:

**Pull request merged**